src/main/java/com/juniorstart/juniorstart/model/UserProfile.java

```diff
@@ -0,0 +1,35 @@
+package com.juniorstart.juniorstart.model;
+
+import lombok.*;
+import javax.persistence.Entity;
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(exclude="userTechnology")
+@Entity
+@Table(name="users_profile")
+@Builder
+public class UserProfile {
+    @Id
+    private UUID privateId;
+
+    @OneToOne
+    @MapsId
+    private User user;
+
+    @Enumerated(EnumType.STRING)
+    private UserRole userRole;
+
+    @ManyToMany(cascade = CascadeType.PERSIST)
+    @JoinTable(name="join_user_technology",
+    joinColumns = {@JoinColumn(name="user_id")},
+    inverseJoinColumns = {@JoinColumn(name="technology_id")})
+    private Set<UserTechnology> userTechnology = new HashSet<>();
```

Comment:
Nie lepiej nazwać technology?

Reply:
a która rzecz dokładniej ?

Reply:
UserTechnology na Technology, sorki za brak zwięzłości :)

Reply:
Ogólnie to myślałem trochę nad nazewnictwem. I stwierdziłem , że pod względem projektu tak będzie lepiej. 
UserTechnology nawiązuje ściśle do użytkownika, a nie wiem jakie rzeczy mógłby być jeszcze wprowadzane etc.

Reply:
Ma to jakiś sens w sumie

Reply:
Ok